package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java

```diff
@@ -203,4 +204,29 @@ private String getAuthenticatorAppMetadata(@NonNull final String pkgName) {
     private boolean isAnAuthenticatorApp(@NonNull final String pkgName) {
         return mPkgAuthenticators.containsKey(pkgName);
     }
+
+    private Signature[] getSignatures(PackageInfo packageInfo) {
```

Comment:
```suggestion
    private Signature[] getSignatures(@Nullable final PackageInfo packageInfo) {
```

Please mark variables `final` where possible, and annotate expectations around null/non-null